.azure/app-cloud-e2e.yml

```diff
@@ -29,6 +29,7 @@ pr:
     include:
       - ".actions/**"
       - ".azure/app-cloud-e2e.yml"
+      - "src/lightning/*.py"
```

Comment:
@Borda This is not working correctly. PL PRs are now running app and store tests. See https://github.com/Lightning-AI/lightning/pull/16926 for example